sdks/python/apache_beam/typehints/typehints_test.py

```diff
@@ -379,9 +379,10 @@ def test_getitem_params_must_be_type_or_constraint(self):
       typehints.Tuple[5, [1, 3]]
     self.assertTrue(e.exception.args[0].startswith(expected_error_prefix))
 
-    with self.assertRaises(TypeError) as e:
-      typehints.Tuple[list, dict]
-    self.assertTrue(e.exception.args[0].startswith(expected_error_prefix))
+    if sys.version_info < (3, 9):
+      with self.assertRaises(TypeError) as e:
+        typehints.Tuple[list, dict]
+      self.assertTrue(e.exception.args[0].startswith(expected_error_prefix))
```

Comment:
Can we add else clauses to these tests that validate that they don't throw?

Reply:
Done